src/git_cache.h

```diff
@@ -1076,8 +1076,8 @@ void git_cache::apply_merge_authorship(commit_tree_buffers &buffers,
                                        parsed_metadata::string_ref cd) {
   buffers.an.append("apple-llvm-mt");
   buffers.cn.append("apple-llvm-mt");
-  buffers.ae.append("mt @ apple-llvm");
-  buffers.ce.append("mt @ apple-llvm");
+  buffers.ae.append("mt@apple-llvm");
+  buffers.ce.append("mt@apple-llvm");
```

Comment:
Why? It still isn't a valid email, so this is just trading gibberish for gibberish

Reply:
Some tools expect a valid email string even if the address itself is invalid.